app/controllers/api/v0/highlights_controller.rb

```diff
@@ -48,6 +48,56 @@ def create
     render json: response_binding, status: :created
   end
 
+  swagger_path '/highlights' do
+    operation :get do
+      key :summary, 'Get the highlight(s)'
+      key :description, 'Get the highlight(s) for the owner.'
```

Comment:
You may want something multiline here, to say that results can be paginated, filtered by various attributes, ordered variously, etc.